topo_processor/stac/collection.py

```diff
@@ -13,7 +13,14 @@
 
 from topo_processor.util import Validity
 
-GLOBAL_PROVIDERS = [pystac.Provider(name="LINZ", description="Land Information New Zealand", roles=["host"])]
+GLOBAL_PROVIDERS = [
```

Comment:
At some point if we need to use a same `provider` in different classes, that would be good to have a dedicated provider definition file.